main.py

```diff
@@ -37,7 +40,7 @@ def reEmail(fname):
 def reAward(fname):
   fhand = open(fname, 'r')
   for line in fhand:
-    extr = re.findall('.*', line)
+    extr = re.findall('-.+', line)
```

Comment:
This will extract all the "- " in front of the award winner. 

`extr = re.findall('- (.*)', line)`